x-pack/plugins/security_solution/server/lib/telemetry/types.ts

```diff
@@ -180,32 +180,35 @@ export interface EndpointMetricsAggregation {
 interface EndpointMetricHits {
   hits: {
     total: { value: number };
-    hits: EndpointMetricDocument[];
+    hits: Array<{ _id: string; _source: EndpointMetricDocument }>;
   };
 }
 
-interface EndpointMetricDocument {
-  _source: {
-    '@timestamp': string;
+export interface EndpointMetricDocument {
+  '@timestamp': string;
+  agent: {
+    id: string;
+    version: string;
+  };
+  Endpoint: {
+    metrics: EndpointMetrics;
+  };
+  elastic: {
     agent: {
       id: string;
-      version: string;
-    };
-    Endpoint: {
-      metrics: EndpointMetrics;
-    };
-    elastic: {
-      agent: {
-        id: string;
-      };
-    };
-    host: {
-      os: EndpointMetricOS;
-    };
-    event: {
-      agent_id_status: string;
     };
   };
```

Comment:
it's the same, but I changed the receiver logic to return the `_source` countent instead of the whole elastic. Check l183: `hits: Array<{ _id: string; _source: EndpointMetricDocument }>;`